var/spack/repos/builtin/packages/upcxx/package.py

```diff
@@ -43,12 +43,15 @@ class Upcxx(Package):
             description="UPC++ cross-compile target (autodetect by default)")
 
     conflicts('cross=none', when='platform=cray',
-              msg='None is unacceptable on Cray. Please specify an appropriate "cross" value')
+              msg='None is unacceptable on Cray.' +
```

Comment:
Without quotes or context "None is unacceptable" seems easy to misread as its English meaning rather than a literal token.